runbotics/runbotics-desktop/src/core/websocket/bpmn/runtime-subscriptions.service.ts

```diff
@@ -55,6 +55,13 @@ export class RuntimeSubscriptionsService {
                     ).behaviour;
                     switch (event.activity.content.type) {
                         case BpmnElementType.ERROR_EVENT_DEFINITION:
+                            const variables = {};
+                            const prevMessages = event.activity.environment.variables['errorMessages'] as string[];
+                            prevMessages.push(desktopTask.output.description);
+                            variables['errorMessages'] = prevMessages;
+                            const setVariables = { ...event.activity.environment.variables, ...variables};
+                            event.activity.environment.assignVariables(setVariables);
+                            processInstanceEvent.error = desktopTask.output.description;
```

Comment:
use this function for it imo https://github.com/runbotics/runbotics/blob/99a3c68536872a7009c2f5918856775fa2be848a/runbotics/runbotics-desktop/src/core/bpm/runtime/runtime.service.ts#L642